test/TestProjects/Unbranded-TypeSpec/src/Generated/UnbrandedTypeSpecClient.cs

```diff
@@ -0,0 +1,2056 @@
+            _keyCredential = credential;
+            _pipeline = HttpPipelineBuilder.Build(options, Array.Empty<HttpPipelinePolicy>(), new HttpPipelinePolicy[] { new AzureKeyCredentialPolicy(_keyCredential, AuthorizationHeader) }, new ResponseClassifier());
+            _endpoint = endpoint;
+            _apiVersion = options.Version;
+        }
+
+        /// <summary> Initializes a new instance of UnbrandedTypeSpecClient. </summary>
+        /// <param name="endpoint"> The Uri to use. </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <param name="options"> The options for configuring the client. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public UnbrandedTypeSpecClient(Uri endpoint, TokenCredential credential, UnbrandedTypeSpecClientOptions options)
+        {
+            Argument.AssertNotNull(endpoint, nameof(endpoint));
+            Argument.AssertNotNull(credential, nameof(credential));
+            options ??= new UnbrandedTypeSpecClientOptions();
+
+            ClientDiagnostics = new TelemetrySource(options, true);
+            _tokenCredential = credential;
+            _pipeline = HttpPipelineBuilder.Build(options, Array.Empty<HttpPipelinePolicy>(), new HttpPipelinePolicy[] { new BearerTokenAuthenticationPolicy(_tokenCredential, AuthorizationScopes) }, new ResponseClassifier());
+            _endpoint = endpoint;
+            _apiVersion = options.Version;
+        }
+
+        /// <summary> top level method. </summary>
+        /// <param name="action"> The DateTime to use. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <include file="Docs/UnbrandedTypeSpecClient.xml" path="doc/members/member[@name='TopActionAsync(DateTimeOffset,CancellationToken)']/*" />
+        internal virtual async Task<Result<Thing>> TopActionAsync(DateTimeOffset action, CancellationToken cancellationToken = default)
+        {
+            RequestContext context = FromCancellationToken(cancellationToken);
+            Result response = await TopActionAsync(action, context).ConfigureAwait(false);
+            return Result.FromValue(Thing.FromResult(response), response);
+        }
+
+        /// <summary> top level method. </summary>
+        /// <param name="action"> The DateTime to use. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <include file="Docs/UnbrandedTypeSpecClient.xml" path="doc/members/member[@name='TopAction(DateTimeOffset,CancellationToken)']/*" />
+        internal virtual Result<Thing> TopAction(DateTimeOffset action, CancellationToken cancellationToken = default)
+        {
+            RequestContext context = FromCancellationToken(cancellationToken);
+            Result response = TopAction(action, context);
```

Comment:
If it's easy, the following will make it easier for humans to look at

```suggestion
            Result result = TopAction(action, context);
```